### ISSUE TITLE
malshare negative result downloaded

### ISSUE THREAD

**Issue by psifertex** [alexandreborges/malwoverview#52]
```
~ ❱ malwoverview -l 1 -L 04af0519d0dbe20bc8dc8ba4d97a791ae3e3474c6372de83087394d219babd47

Sample not found by the provided hash.



MALWARE SAMPLE SAVED!
~ ❱ file 04af0519d0dbe20bc8dc8ba4d97a791ae3e3474c6372de83087394d219babd47
04af0519d0dbe20bc8dc8ba4d97a791ae3e3474c6372de83087394d219babd47: ASCII text, with no line terminators
~ ❱ cat 04af0519d0dbe20bc8dc8ba4d97a791ae3e3474c6372de83087394d219babd47
Sample not found by hash (04af0519d0dbe20bc8dc8ba4d97a791ae3e3474c6372de83087394d219babd47)⏎
```

**Comment by alexandreborges** [comment]
Good afternoon.

This issue has already fixed, and it will be included in the version 6.0.1.

Have a good day.

**Comment by alexandreborges** [comment]
Jordan @psifertex , 

As this fix has already included into version 6.0.1, I am closing this issue.

Have an excellent day.
